release/models/policy-forwarding/openconfig-pf-forwarding-policies.yang

```diff
@@ -270,6 +276,14 @@ submodule openconfig-pf-forwarding-policies {
         packets that match the rule.";
     }
 
+    leaf accept {
+      type boolean;
+      default false;
```

Comment:
I don't agree that the default should be false -- it will mean that any existing policy-forwarding rule will automatically start dropping packets, which is going to be harmful. We should consider:

 * having no default -- since we've found that having a default value creates an implicit specification.
 * making this a `drop` leaf since it seems more common that one would want to forward a packet vs. drop it.